### PR TITLE
Create (initial) 6 month organization plan

### PR DESCRIPTION
**Pull request by zachlatta**
I figure having something short and (more) specific is better than having what we currently have. I'd recommend merging this for the time being (unless you see any issues) and we can improve it as we progress. I know I'll be making more changes.

### CODE REVIEW

File: GOALS.md [hunk 1]
```diff
@@ -1,225 +1,22 @@
...
+- [ ] First two meetings are planned out and tested in real life
+    - We should be able to concretely answer the question "how do I run the
+      first two meetings" from club leaders
+
+# January - April
+
+- #1 focus :: weekly club member growth
+    - It's important that we have a mechanism set up to measure this (maybe
+      weekly calls with every club?)
+- Goal: start 50 clubs by the end of February
```
Comment (zachlatta) [inline GOALS.md]: @hackclub/staff what are your thoughts on this goal?

Comment (MaxWofford) [inline GOALS.md]: 50 clubs seems doable.

Comment (paked) [inline GOALS.md]: Lets do it!

Comment (zachlatta) [inline GOALS.md]: Awesome, let's do it :-).

**Comment by jonleung** [comment]
:+1:

**Pull request merged**